service/payments/src/main/java/nl/tudelft/wdm/group1/payments/events/Producer.java

```diff
@@ -21,7 +21,7 @@
 
     public void emitPaymentCreated(Payment payment) {
         logger.info("Producing [{}] -> {}", PaymentsTopics.PAYMENT_CREATED, payment);
-        this.kafkaTemplate.send(PaymentsTopics.PAYMENT_CREATED, payment);
+        this.kafkaTemplate.send(PaymentsTopics.PAYMENT_CREATED, payment.getUserId().toString(), payment);
```

Comment:
This topic is not partitioned right? Only the topics which are in `UsersTopics` are.

Reply:
~Ahh op die fiets~ Good catch, I'll add a Bean config

Reply:
(apologies for not catching this myself, my local Kubernetes configuration has acted up lately and I have tried but failed repeatedly to test this myself)